### ISSUE TITLE
Base call to method shall be surrounded with empty lines

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1887]
A base call to a method shall be surrounded with empty lines, so that it is easily to spot.

Following should trigger a warning:
```csharp
public override void DoSomething()
{
   DoSomethingElse();
   base.DoSomething();
   DoAnything();
}
```

It shall be fixed to:
```csharp
public override void DoSomething()
{
   DoSomethingElse();

   base.DoSomething();

   DoAnything();
}
```
